depends/packages/rapidcheck.mk

```diff
@@ -0,0 +1,21 @@
+package=rapidcheck
+
+$(package)_version:1.0
+
+$(package)_download_path:https://github.com/Christewart/rapidcheck/releases/download/1.0
+
+$(package)_file_name:rapidcheck-1.0.tar.gz
```

Comment:
You should be able to use something like  `$(package)-$($(package)_version)` here.